manim/mobject/geometry.py

```diff
@@ -807,22 +850,98 @@ def __init__(self, *args, **kwargs):
 
     @property
     def base(self):
+        r"""The base point of the arrow tip.
+        
+        This is the point connecting to the arrow line.
+
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 0, 0]), buff=0)
+            >>> arrow.tip.base.round(2)
+            array([1.65, 0.  , 0.  ])
+        
+        """
         return self.point_from_proportion(0.5)
 
     @property
     def tip_point(self):
+        r"""The tip point of the arrow tip.
+        
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 0, 0]), buff=0)
+            >>> arrow.tip.tip_point.round(2)
+            array([2., 0., 0.])
+        
+        """
         return self.points[0]
 
     @property
     def vector(self):
+        r"""The arrow tip vector spanning from base to tip.
+        
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 2, 0]), buff=0)
+            >>> arrow.tip.vector.round(2)
+            array([0.25, 0.25, 0.  ])
+        
+        """
         return self.tip_point - self.base
 
     @property
     def tip_angle(self):
+        r"""The angle of the arrow tip.
+        
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([1, 1, 0]), buff=0)
+            >>> round(arrow.tip.tip_angle, 5) == round(PI/4, 5)
+            True
+        
+        """
         return angle_of_vector(self.vector)
 
+    @property
+    def tip_length(self):
+        r"""The length of the arrow tip.
+
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([1, 2, 0]))
+            >>> round(arrow.tip.tip_length, 3)
+            0.35
+
+        """
+        return get_norm(self.vector)
+
 
 class ArrowFilledTip(ArrowTip):
+    r"""Base class for arrow tips with filled tip.
+
+    Note
+    ----
+    In comparison to :class:`ArrowTip`, this class only provides
+    different default settings for styling arrow tips. These settings
+    (in particular `fill_opacity` and `stroke_width`) can also be
+    overridden manually.
+
```

Comment:
Suggestion: Maybe add an Examples section that says something along the lines of "using ArrowFilledTip is a shorthand to using ArrowTip(...)`.

Reply:
I like that! 5528cc30a351ff57a5b06f3ab085620148cc2fe8